src/checkoutservice/main.go

```diff
@@ -109,6 +132,14 @@ func initStats(exporter *stackdriver.Exporter) {
 }
 
 func initTracing() {
+	// This is a demo app with low QPS. trace.AlwaysSample() is used here
+	// to make sure traces are available for observation and analysis.
+	// In a production environment or high QPS setup please use
+	// trace.ProbabilitySampler set at the desired probability.
+	trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
```

Comment:
Do we have to set this on all services? I assumed sample=y/n is enough to set on the frontend, which all traces to this service originate from.